### PR TITLE
Fix comparing uuids of elements recycler list 

### PR DESCRIPTION
**Pull request by osdnk**
Fixes RNBW-####

## What changed (plus any additional context for devs)

https://github.com/Flipkart/recyclerlistview/blob/master/docs/guides/samplecode/README.md
image

This allows for reducing the offset a bit what helps in performance in general 

## PoW (screenshots / screen recordings)

Before: 

## Dev checklist for QA: what to test

https://user-images.githubusercontent.com/25709300/151558714-3253a628-d0dc-4f77-a73c-b69137610d18.mov

After: 

## Final checklist

https://user-images.githubusercontent.com/25709300/151558734-b763b30d-24de-4203-9117-112b3c7a7365.mov

### CODE REVIEW

File: src/components/asset-list/RecyclerAssetList2/core/RawRecyclerList.tsx [hunk 1]
```diff
@@ -52,7 +52,7 @@ const RawMemoRecyclerAssetList = React.memo(function RawRecyclerAssetList({
       onLayout={onLayout}
       ref={ref as LegacyRef<RecyclerListViewRef>}
       refreshControl={<RefreshControl />}
-      renderAheadOffset={ios ? 700 : 2000}
```
Comment (terrysahaidak) [inline src/components/asset-list/RecyclerAssetList2/core/RawRecyclerList.tsx:55]: I didn't find much of a difference between 700 and 1200, as an example, on my phone, if you scroll fast enough you can get blank spaces but it's kinda ok.

**Review by terrysahaidak — APPROVED** [review]
well yeah, funny no one caught that dataProvider comparison thing.

File: src/components/asset-list/RecyclerAssetList2/core/RawRecyclerList.tsx [hunk 2]
```diff
@@ -11,7 +11,7 @@ import useLayoutItemAnimator from './useLayoutItemAnimator';
 import { useCoinListEdited } from '@rainbow-me/hooks';
 
 const dataProvider = new DataProvider((r1, r2) => {
-  return r1.uid === r2.uid;
+  return r1.uid !== r2.uid;
```
Comment (skylarbarrera) [inline src/components/asset-list/RecyclerAssetList2/core/RawRecyclerList.tsx:14]: lmaoo

**Review by skylarbarrera — APPROVED** [review]
:)

**Pull request merged**